### ISSUE TITLE
Feature Request: Autostart and Remember my password

### ISSUE THREAD

**Issue by mattcoxonline** [azirevpn/azclient#35]
I want the option in the Azire client to remember my password, and to automatically start up and sign in with my computer.

Every time I restart the computer, I have to start up the AZClient manually, then type in the password and hit connect. Sometimes I forget to do it. By this time, the computer has probably sent loads of traffic to my ISP and I want to prevent that.

I want the AzireVPN client "Always on" or as close as I can get to this. As of today the client requires several manual processes before it's connected.

**Comment by dm17** [comment]
Yes, the "remember my password" doesn't work for me either. Maybe because I'm not quitting gracefully? But there's no way to do it on a tiling windows manager (I see no quit button so I cntl^c).
